components/navbar/Navbar.js

```diff
@@ -26,13 +17,21 @@ function Navbar() {
             variant="h6"
             flexGrow={0}
           />
-          <MobileMenu pages={pages} loggedIn={!!currentUser} />
+          <MenuManager
+            version="mobile"
+            loggedIn={!!currentUser}
```

Comment:
Could be better to use Boolean(currentUser) ?

Reply:
Good idea, didn't know that method